frontend/tests/lighthouse.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { mkdir } from "fs/promises";
+import { join } from "path";
+import { playAudit } from "playwright-lighthouse";
+import { test } from "@playwright/test";
+//import { paths } from "./axe.spec";
+import analyses from "@/fixtures/analyses.json" with { type: "json" };
+import { log } from "./util";
+
+log();
+
+export const paths = [
+  "/testbed",
+  "/", 
+  "/load-analysis", 
+  "/new-analysis",
+  "/about",
+  ...analyses.map((analysis) => `/analysis/${analysis.id}`),
+];
+
+const thresholds = {
+  performance: 80,
+  accessibility: 90,
+  "best-practices": 85,
+  seo: 85,
+  pwa: 50,
+};
+
+const config = {
+  extends: "lighthouse:default",
+  settings: {
+    formFactor: "desktop" as const,
+    screenEmulation: { disabled: true },
+    //emulatedUserAgent: "desktop",
+    // Enable APCA contrast checking
+    accessibilityScoring: "apca",
+  },
+};
+
+const port = 1234;
+
+// Ensure reports directory exists
+const reportsDir = "lighthouse-reports";
+await mkdir(reportsDir, { recursive: true });
```

Comment:
This shouldn't be needed. Typically this type of low level thing would be handled by the library exporting the file. I tested removing this line and starting with no folder, and the `playwright-lighthouse` library was able to create the folder itself.